src/pybind/mgr/dashboard/frontend/src/app/app.component.scss

```diff
@@ -6,26 +6,110 @@
   padding: 0;
 }
 
-::ng-deep #toast-container {
-  margin-top: 2vw;
+::ng-deep .overlay-container {
+  #toast-container {
+    margin-top: 2vw;
 
-  @media (max-width: 1600px) {
-    margin-top: 2.5vw;
-  }
+    @media (max-width: 1600px) {
+      margin-top: 2.5vw;
+    }
 
-  @media (max-width: $screen-md-max) {
-    margin-top: 9vw;
-  }
+    @media (max-width: $screen-md-max) {
+      margin-top: 9vw;
+    }
+
+    @media (max-width: 900px) {
+      margin-top: 10vw;
+    }
+
+    @media (max-width: 319px) {
+      margin-top: 11vw;
+    }
 
-  @media (max-width: 900px) {
-    margin-top: 10vw;
+    @media (max-width: 260px) {
+      margin-top: 14vw;
+    }
   }
 
-  @media (max-width: 319px) {
-    margin-top: 11vw;
+  &.block-ui {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    left: 0;
+    overflow-x: hidden;
+    overflow-y: auto;
+    width: 100%;
+    height: 100%;
+    z-index: 1050;
+    background-color: rgba(0, 0, 0, 0.5);
+
+    #toast-container {
+      position: initial;
+      margin: 0;
+
+      cd-toast {
+        margin: 1.75rem auto;
+        padding: 1rem;
+        max-width: 500px;
+        color: #212529;
+        background-color: #fff;
+        border: 1px solid rgba(0, 0, 0, 0.2);
+        border-radius: 0.3rem;
+        outline: 0;
+        -webkit-box-shadow: none;
+        -moz-box-shadow: none;
+        box-shadow: none;
+
+        .toast-title {
```

Comment:
Ahh yeah this might would make sense. I just looked up the background color of the dialog content, because at that time I had no intention to also style the title of the dialog. After I did style the title I did not double check whether the title uses another background color or not :)